### PR TITLE
Fix reference doc links

### LINKED ISSUE #369: Some links are broken
Some links in [the official document about data_augmentor](https://docs.blueoil.org/reference/data_augmentor.html) are broken.

- lmnet.data_augmentor.Blur
- lmnet.data_augmentor.Brightness
- lmnet.data_augmentor.Color
- lmnet.data_augmentor.Contrast

### PR DESCRIPTION
**Pull request by Joeper214**
## Motivation and Context

Some links in docs on references are broken. This PR will fix then.

Fix #369 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: lmnet/lmnet/data_augmentor.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Blur(data_processor.Processor):
     """Gaussian blur filter.
 
     Reference:
-        http://pillow.readthedocs.io/en/4.3.x/Referenceserence/ImageFilter.html#PIL.ImageFilter.GaussianBlur
+        https://pillow.readthedocs.io/en/stable/reference/ImageFilter.html#PIL.ImageFilter.GaussianBlur
```
Comment (iizukak) [inline lmnet/lmnet/data_augmentor.py]: I thinks document of current version is better than latest `stable` version.
Current using version's documentation is not avairable?

Comment (Joeper214) [reply lmnet/lmnet/data_augmentor.py]: As far as I know, Blueoil's current master branch is using 

[`Pillow==6.0.0`](https://github.com/blue-oil/blueoil/blob/9fcdbfc8808ef8f4509e0c4ae0729b30c298e943/lmnet/requirements.txt#L6)

However, if you select a specific version of the documentation, it doesn't exist anymore. The selection of documentation is either the latest or stable release version.

Comment (Joeper214) [reply lmnet/lmnet/data_augmentor.py]: As far as I know, Blueoil's current master branch is using

`Pillow==6.0.0`

However, if you select a specific version on the documentation, it doesn't exist anymore. The selection of documentation is either the latest or stable release version.

So selecting version 6.0.0 is kind of impossible. As it is not active in the docs.

https://readthedocs.org/projects/pillow/versions/

Comment (iizukak) [reply lmnet/lmnet/data_augmentor.py]: @Joeper214 Oh, thanks.
We should update Pillow...

**Pull request merged**